app/services/invoices/customer_usage_service.rb

```diff
@@ -86,7 +86,7 @@ def add_charge_fees
     end
 
     def charge_usage(charge)
-      return charge_usage_without_cache(charge) if organization.clickhouse_aggregation?
```

Comment:
Will we support all aggregations in CH? I was thinking it was also good to check if the billable metric supports CH?

Reply:
all `billable_metrics`  should support Clickhouse yes, it's difficult to have an hybrid event store (between PG and Clickhouse) at the same time, mainly due to ingestion